### DISCUSSION TITLE
Welcome to ipfilter Discussions!

### DISCUSSION THREAD

**Discussion by DavidMoore** [DavidMoore/ipfilter#76] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
thanks for the program. I see it blocked the Xunlei IP's on my utorrent client. nice

**Reply by winnie23-23** [reply]
Can ipfilter list include some Chinese IPs causing a lot of havoc like in this thread?

https://github.com/anacrolix/torrent/issues/889
